Packages/vcs/Lib/configurator.py

```diff
@@ -510,12 +447,13 @@ def reset_template_changes(state):
           logo_button.set_state(1)
 
     def setup_animation(self):
-        if self.initialized == False:
+        if self.initialized is False and [True for d in self.displays if d.g_type not in ("marker", "text", "textcombined", "fillarea", "line", "textorientation", "texttable")]:
```

Comment:
It would help. It would be easier to read and self-documenting to use a `display_supports_animation` method or similar.